### DISCUSSION TITLE
Possibility of connecting BingAI (gpt 4 + websearch)

### DISCUSSION THREAD

**Discussion by FlyingHermes** [ChatGPTNextWeb/NextChat#3890] [Q&A]
I have previously asked a similar question here and received an answer regarding the process of connecting BingAI to ChatgptNextWeb. However, I didn't fully comprehend the implementation steps. Could you kindly assist me and the entire community?

Repository: Harry-zklcdc/go-proxy-bingai#301

![изображение](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/79836672/fb497479-ad8d-42b0-ac16-acafcf4f08a5)

**Comment by Roelauph** [comment]
Add chatcmpl-NewBing in Custom Models.
